docs/_basic/authenticating_oauth.md

```diff
@@ -27,7 +68,7 @@ const app = new App({
   clientId: process.env.SLACK_CLIENT_ID,
   clientSecret: process.env.SLACK_CLIENT_SECRET,
   stateSecret: 'my-state-secret',
-  scopes: ['channels:read', 'groups:read', 'channels:manage', 'chat:write', 'incoming-webhook'],
+  scopes: ['chat:write', 'incoming-webhook'],
```

Comment:
Doesn't need so many scopes. It's already a very cluttered code sample.

Reply:
I agree. If we revisit this part, we may want to have `commands` and `chat:write` instead because that's a common minimum set for Bolt apps (you use `say()`, shortcuts, slash commands etc. but don't use incoming webhooks with Bolt, right?)

Reply:
Sure thing - I've modified the scopes and also adjusted the comments in the `storeInstallation` bit.